packages/backend/src/prisma/schema.prisma

```diff
@@ -38,10 +38,12 @@ model Invoice {
   id          String   @id @default(cuid())
   invoiceDate DateTime @default(now())
 
-  createdByUserId String
-  customerAddress String?
-  organizationId  String
-  customerName    String
+  createdByUserId  String
+  customerAddress  String?
+  organizationId   String
+  customerName     String
+  invoiceWorkFrom  DateTime?
+  invoiceWorkUntil DateTime?
```

Comment:
I don't remember our discussions, but should these be optional?

Reply:
Does that mean they will eventually be required?
If so the changes here will affect all the validations and backend implementation etc.

I would then suggest to start with a clean migration from this ticket.